src/modules/dashboard/components/Community/Community.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { FC, useCallback, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { Address } from '~types/index';
+import {
+  useColonySubscribedUsersQuery,
+  useLoggedInUser,
+  useUserColonyAddressesQuery,
+  useSubscribeToColonyMutation,
+  ColonySubscribedUsersDocument,
+} from '~data/index';
+import { domainsAndRolesFetcher } from '../../fetchers';
+import { getCommunityRoles } from '../../../transformers';
+import { useDataFetcher, useTransformer } from '~utils/hooks';
+import { ROLES_COMMUNITY } from '~constants';
+import { sortObjectsBy } from '~utils/arrays';
+
+import { Table, TableBody, TableCell } from '~core/Table';
+import { SpinnerLoader } from '~core/Preloaders';
+import UserListItem from '~admin/UserListItem';
+import Button from '~core/Button';
+
+import styles from './Community.css';
+
+const MSG = defineMessages({
+  loading: {
+    id: 'dashboard.Community.loading',
+    defaultMessage: "Loading Colony's users...",
+  },
+  callToSubscribe: {
+    id: 'dashboard.Community.callToSubscribe',
+    defaultMessage: `{noMembers, select,
+      true {No members yet. {action} to become the first member of this colony.}
+      other {{action} to become a member of this colony.}
+    }`,
+  },
+  subscribe: {
+    id: 'dashboard.Community.subscribe',
+    defaultMessage: `Subscribe now`,
+  },
+  subscribedReward: {
+    id: 'dashboard.Community.subscribedReward',
+    defaultMessage: `{star} Subscribed!`,
+  },
+});
+
+interface Props {
+  colonyAddress: Address;
+}
+
+const Community = ({ colonyAddress }: Props) => {
```

Comment:
`displayName` please!

Reply:
Granted it's wrong, and an older format introduced by copy-paste, but it is there...

https://github.com/JoinColony/colonyDapp/blob/31e41bdb30de8a4b21461ffe58daaa861a0bbd31/src/modules/dashboard/components/Community/Community.tsx#L209